### PR TITLE
DOCS-11217 slight rephrasing

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/persistence-gateway.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 = Persistence Gateway
 
-To store state across applications, Anypoint Runtime Fabric provides Persistence Gateway. Persistence Gateway enables Mule applications deployed to a Mule runtime engine (Mule) to store and share data across Mule application replicas and application restarts.
+Anypoint Runtime Fabric provides Persistence Gateway, which enables Mule applications deployed to a Mule runtime engine to store and share data across application replicas and restarts.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/persistence-gateway.adoc]: a Mule runtime engine > a Mule instance

Comment (IsaacEldridge) [reply modules/ROOT/pages/persistence-gateway.adoc]: Fixed.

File: modules/ROOT/pages/persistence-gateway.adoc [hunk 2]
```diff
@@ -11,7 +11,7 @@ Persistence Gateway is supported on Anypoint Runtime Fabric 1.9 or later and Mul
 
 After Persistence Gateway is configured in Anypoint Runtime Fabric, it is available for Mule applications deployed to Mule runtime engine, version 4.2.1 or later. When configured, users can select *Use Persistent Object Storage* when deploying an application using Runtime Manager. See xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric] for more information.
 
-Mule applications use the Object Store REST API to connect to Persistence Gateway. This enables you to deploy to both Anypoint Runtime Fabric and CloudHub without having to modify your Mule application. 
+Mule applications use the Object Store v2 REST API via the Mule Object Store Connector to connect to Persistence Gateway. This enables you to deploy to both Anypoint Runtime Fabric and CloudHub without having to modify your Mule application.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/persistence-gateway.adoc]: Remove **Mule** before Object Store Connector.

**Review by Cristian-Venticinque — DISMISSED** [review]
Approved with comments

File: modules/ROOT/pages/persistence-gateway.adoc (hunk 2 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/persistence-gateway.adoc]:
```suggestion
Mule applications use the Object Store v2 REST API via the Object Store Connector to connect to Persistence Gateway. This enables you to deploy to both Anypoint Runtime Fabric and CloudHub without having to modify your Mule application. 
```

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**